Assets/Scripts/GridManager.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GridManager : MonoBehaviour
+{
+    // Start is called before the first frame update
```

Comment:
For future reference, we should remove any empty methods and comments that don't provide any real value to the code